basic-application/basic-application-webapp/src/main/java/fr/openwide/core/basicapp/web/application/BasicApplicationApplication.java

```diff
@@ -105,7 +110,9 @@ protected IConverterLocator newConverterLocator() {
 		converterLocator.set(TechnicalUser.class, UserRenderer.get());
 		converterLocator.set(BasicUser.class, UserRenderer.get());
 		converterLocator.set(UserGroup.class, UserGroupRenderer.get());
+		converterLocator.set(GenericLocalizedGenericListItem.class, LocalizedGenericListItemRenderer.get());
```

Comment:
As far as I know, it's no use doing that: the `ConverterLocator` doesn't handle inheritance. You must set the converter for each and every instantiable subclass of `GenericLocalizedGenericListItem`.
Maybe it's something we should add to the wiki page about renderers.